### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by rajputvishwas**
Added an alternate approach to setup 3scale APIcast

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
 oc new-app -f https://raw.githubusercontent.com/3scale/apicast/master/openshift/apicast-template.yml
 ```
 
+### OKD (Openshift-origin 3.11)
```
Comment (porueesq) [inline README.md:57]: What does `OKD` mean? Perhaps this can be expanded, and then use the abbreviated form in the text.

File: README.md [hunk 2]
```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
...
+
+There are multiple approaches to setup 3scale APIcast on Openshift origin. The one with yaml file is the common approach to setup 3scale account. Another approach is the easiest one where you need to pull 3scale image on local docker repository, then create new application in OKD from docker image by passing required parameters. I have used this approach to setup 3scale APIcast on my all in one OKD cluster on AWS.
```
Comment (porueesq) [inline README.md:59]:
```suggestion
Using yaml files is the common approach to configure 3scale APIcast on Openshift origin. Another approach is pulling a 3scale image on the local docker repository, then creating a new application in OKD from the Docker image by passing required parameters. For example, this latter approach was used to setup 3scale APIcast in OKD cluster on AWS.
```

File: README.md [hunk 3]
```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
...
+
+- Pull the 3scale image to your local docker repository.
```
Comment (porueesq) [inline README.md:61]:
```suggestion
- Pull the 3scale image to the local docker repository:
```

File: README.md [hunk 4]
```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
...
+
+docker pull registry.access.redhat.com/3scale-amp20/apicast-gateway:1.0
+
+In all cases, in order to setup 3scale you need to have 3scale account. Steps to create and configure 3scale account are given here (https://github.com/tnscorcoran/rhsso-3scale).
```
Comment (porueesq) [inline README.md:65]: Can you relocate this in a subsection called _Prerequisites_?

File: README.md (hunk 3 above)
Comment (porueesq) [reply README.md:61]: If this is the first step, place it under a section called _Procedure_.

File: README.md [hunk 5]
```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
...
+
+- Once configured 3scale account, you need 'ACCESS_TOKEN' along with your 3scale account URL which is called `ADMIN_PORTAL_DOMAIN`, to create secret which in turn used by your application for authentication. Create secret as discussed in the above approach.
```
Comment (porueesq) [inline README.md:67]: _Create secret as discussed in the above approach._ --> Is this a prerequisite? If not, please reword it to remove the term "above"

File: README.md [hunk 6]
```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
...
+
+oc create secret generic apicast-configuration-url-secret --from-literal=password=https://2040edXXXXXXXXX80d6@XXXXX-admin.3scale.net --type=kubernetes.io/basic-auth
+
+Using 'ACCESS_TOKEN' and 'ADMIN_PORTAL_DOMAIN', you can configure another environment variable 'THREESCALE_PORTAL_ENDPOINT'.
```
Comment (porueesq) [inline README.md:71]:
```suggestion
. Using 'ACCESS_TOKEN' and 'ADMIN_PORTAL_DOMAIN', you can configure another environment variable 'THREESCALE_PORTAL_ENDPOINT':
```

File: README.md [hunk 7]
```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
...
+
+THREESCALE_PORTAL_ENDPOINT=https://ACCESS_TOKEN@ADMIN_PORTAL_DOMAIN
+
+- In order to create application using local 3scale docker image, you need to pass environment variables
```
Comment (porueesq) [inline README.md:75]:
```suggestion
- Pass the environment variables to create an application using local 3scale Docker image:
```

File: README.md (hunk 1 above)
Comment (mikz) [reply README.md:57]: https://www.okd.io

I don't see where they explain what it stands for.

File: README.md [hunk 8]
```diff
@@ -54,6 +54,30 @@ oc create secret generic apicast-configuration-url-secret \
...
+
+oc new-app --docker-image=registry.access.redhat.com/3scale-amp20/apicast-gateway:1.0  --param CONFIGURATION_URL_SECRET=apicast-configuration-url-secret -e THREESCALE_PORTAL_ENDPOINT=https://2040edXXXXXXXXX80d6@XXXXX-admin.3scale.net
+
+This should setup 3scale APIcast on your all in one OKD cluster.
```
Comment (porueesq) [inline README.md:79]: Is there a visible way to the user to show that the configuration is complete?

Comment (rajputvishwas) [reply README.md:86]: I used to check service hit count  in 3scale accout

Comment (porueesq) [reply README.md:86]: Perhaps you can include a verification step, so the user can confirm that the config is complete.

File: README.md (hunk 1 above)
Comment (porueesq) [reply README.md:57]: In this case, the heading should be reworded. Suggestions: 

- Configuring 3scale APIcast with OKD
- Using OKD to configure 3scale APIcast

Then, the first sentence should define OKD: 
_OKD is a distribution of Kubernetes optimized for continuous application development and multi-tenant deployment._

**Comment by eguzki** [comment]
closing for inactivity
